tempesta_fw/http.c

```diff
@@ -3185,12 +3357,9 @@ tfw_cfgop_resp_body(TfwCfgSpec *cs, TfwCfgEntry *ce)
 		return -EINVAL;
 	}
 
-	if (tfw_cfgop_parse_http_status(ce->vals[0], &code))
-	{
+	if (tfw_cfgop_parse_http_status(ce->vals[0], &code)) {
 		TFW_ERR_NL("Unable to parse 'response_body' value: '%s'\n",
-			   ce->vals[0]
-			   ? ce->vals[0]
-			   : "No value specified");
+			   ce->vals[0]);
```

Comment:
Can't this message be made clearer? What the 'value'? Perhaps, 'HTTP status code'?

Reply:
Corrected.